text/0000-utilities-profile.md

```diff
@@ -0,0 +1,42 @@
+- Start Date: 2015-12-03
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This proposes we move `extend`, `select`, and `option` from the base
+specification to its own profile.
+
+Note that I'm calling this a "profile" versus a "namespace." This is pending
+this [RFC](https://github.com/refractproject/rfcs/pull/30). If we do not go the
+route proposed there, we can refer to this as a Utility Namespace.
+
+# Motivation
+
+The main motivation is to make the base specification as simple as possible. In
+ever implementation we've created for consuming Refract, such as from the Parse
+Result or API Description results, we have not needed to use any of these
```

Comment:
BUT I'm all for leaving in the spec as well. That's really the topic of discussion, as to whether they are needed for every possible profile/namespace. If we think so, let's close this RFC. 

Note: you @zdne have mentioned we should address inheritance in the base spec, and if so, these element may become more important. So far, we have left the inheritance up to the Data Structure spec, which is really the place these elements are needed.